docker-assets/httpd-environment.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ConditionPathExists=/etc/container-environment
```

Comment:
What environment variables need to be provided to the httpd service?

The only one I see in the other PR is `HTTPD_AUTH_TYPE` is that used by httpd itself?

Reply:
Spoke with @abellotti, this is for future use as some of the config files will require environment variable interpolation.